mini_pupper_recognition/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>mini_pupper_recognition</name>
```

Comment:
Please think about what are the core functionality/capabilities mini_pupper_ros repo shall provide. Let's think about moving applications like dance, maze runner into a folder like demos/example/application. 

reference turtlebot3_example
![Uploading Screenshot 2024-08-13 at 12.05.09 AM.png…]()